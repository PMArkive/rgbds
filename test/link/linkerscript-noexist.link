INCLUDE "linkerscript-noexist.inc"
ROM0
	"test"
