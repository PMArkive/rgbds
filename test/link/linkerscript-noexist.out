error: linkerscript-noexist.link(1): Failed to open included linker script "linkerscript-noexist.inc": No such file or directory
Linking failed with 1 error
